server/reports/commission/commission.js

```diff
@@ -189,17 +201,15 @@ export class CommissionReport {
       for (let i = 0; i < sales.length; i++) {
         var items = sales[i].PurchasedItems;
         for (let j = 0; j < items.length; j++) {
-          if (items[j].BarcodeId)
+          if (items[j].BarcodeId) {
             productIDs.add(items[j].BarcodeId);
-          else
+          } else {
             serviceIDs.add(items[j].Id);
+          }
         }
       }
     }
-    return Promise.all([
-      this.requestProducts(Array.from(productIDs)),
-      this.requestServices(Array.from(serviceIDs))
-    ]);
+    return Promise.all([this.requestProducts(Array.from(productIDs)), this.requestServices(Array.from(serviceIDs))]);
```

Comment:
I think `array-element-newline` should be disabled. If
```
return this.requestSales()
  .then(this.requestSaleItems.bind(this))
  .then(this.requestVisits.bind(this))
  .then(this.merge.bind(this));
```
is acceptable, then multi-line lists should as well.